cmd/crc/cmd/version.go

```diff
@@ -30,8 +30,8 @@ func init() {
 
 var versionCmd = &cobra.Command{
 	Use:   "version",
-	Short: "Get CodeReady Container version.",
-	Long:  "Get the CodeReady Container version currently installed.",
+	Short: "Print the version of crc",
+	Long:  "Print the version of crc",
```

Comment:
`crc` or `CRC` ?

Reply:
Personally, I interpret "crc" as the literal binary `crc`. "CRC" refers to the entirety of the project, not just the binary itself.

Reply:
I interpret crc and CRC the same way. And given that the version command prints out both the version of the crc binary and the version of the OpenShift cluster
$ crc version
version: 1.0.0-dev+e52aead
OpenShift version: 4.2.0-rc.0 (not embedded in binary)
$
my suggestion doesn't look right.

openshift-install simply says "Print version information" here.

Reply:
$ crc version
crc version: 1.0.0-dev+e52aead
^^^ needs to be here I think

Reply:
CRC is the project
`crc` is the command